src/controllers/dynakube/dynakube_controller.go

```diff
@@ -215,21 +215,21 @@ func (controller *DynakubeController) reconcileDynaKube(ctx context.Context, dkS
 
 	if dkState.Instance.HostMonitoringMode() {
 		upd, err = oneagent.NewOneAgentReconciler(
-			controller.client, controller.apiReader, controller.scheme, dkState.Instance, daemonset.HostMonitoringFeature,
+			controller.client, controller.apiReader, controller.scheme, dkState.Instance, deploymentmetadata.DeploymentTypeHostMonitoring,
```

Comment:
I like the reuse of the constant, but I think it should be the other way around.
`deploymentmetadata` should use the constants defined in `oneagent/daemonset/config.go` (or somewhere around the oneagent reconciler)

Reply:
We can not do that, as deploymentmetadata reports to ClusterDataHub, where those constants have expected values, if we change the value of them, I guess our reporting to ClusterDataHub is not working anymore

Reply:
the constants would have the same values, just be in a different location?
I'd like the `deploymentmetadata` package to reuse constants that exist in the appropriate packages and not have `deploymentmetadata` spread around the whole code base.

Reply:
Moving them out of `deploymentmetadata` would tie them to the package they are moved into, but actually those consts are tied to the deploymentmetadata strongly as a change in them would result in loosing reports ind CDH